docs/check-conformance-test-suite-docs.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function init_settings() {
+  TEST_SUITE_DOCUMENT="unchecked"
+
+  STABLE_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
+  echo "STABLE_VERSION: $STABLE_VERSION"
+  
+  RELEASE_VERSION="$(echo "$STABLE_VERSION" | awk -F '.' '{print $1 "." $2}').0"
+  echo "RELEASE_VERSION: $RELEASE_VERSION"
+  
+  KUBE_CONFORMANCE_FILENAME="KubeConformance-$(echo "$STABLE_VERSION" | awk -F '.' '{print $1 "." $2}' | sed 's/v//').md"
+  echo "KUBE_CONFORMANCE_FILENAME: $KUBE_CONFORMANCE_FILENAME"
+  
+  CONFORMANCE_BASE_URL="${CONFORMANCE_BASE_URL:-https://raw.githubusercontent.com/cncf/k8s-conformance/master/docs}"
+  echo "CONFORMANCE_BASE_URL: $CONFORMANCE_BASE_URL"
+  
+  MD_FILE_URL=${CONFORMANCE_BASE_URL}/${KUBE_CONFORMANCE_FILENAME}
+  echo "MD_FILE_URL: $MD_FILE_URL"
+  
+  RESPONSE_CODE=$(curl -s -o /dev/null "$MD_FILE_URL" --write-out "%{http_code}")
+  if [[ $RESPONSE_CODE ==  "404" ]]; then
+    TEST_SUITE_DOCUMENT="missing"
+  fi
+}
+
+function setup_k8s_repo() {
+  mkdir -p /tmp/go/src/k8s.io
+  cd "$_"
+  git clone --depth 1 --branch "${RELEASE_VERSION}" https://github.com/kubernetes/kubernetes.git
+  export KUBE_ROOT=/tmp/go/src/k8s.io/kubernetes
+  cd "${KUBE_ROOT}"
+  git log -1
+}
+
+function install_go() {
+  cd ${KUBE_ROOT}
+  GO_VERSION=$(grep -A1 "golang: upstream version" build/dependencies.yaml | tail -1 | awk -F ':' '{print $2}' | sed 's/^ *//')
+  echo "GO_VERSION: $GO_VERSION"
+  curl -L https://dl.google.com/go/go"${GO_VERSION}".linux-amd64.tar.gz \
+    | sudo tar --directory /usr/local --extract --ungzip
+  export PATH="$PATH:/usr/local/go/bin"
+  go version
+}
+
+function build_deps() {
+  sudo apt install -y gcc
+  cd "${KUBE_ROOT}"
+  ./hack/make-rules/build.sh vendor/github.com/onsi/ginkgo/ginkgo test/e2e/e2e.test
+}
+
+function build_release_documentation() {
+  cd "${KUBE_ROOT}"
+  ./_output/bin/ginkgo --dry-run=true --focus='[Conformance]' ./_output/bin/e2e.test -- --spec-dump "${KUBE_ROOT}/_output/specsummaries.json" > /dev/null
+
+  DOC_VERSION="$(echo "$STABLE_VERSION" | awk -F '.' '{print $1 "." $2}' | sed 's/v//')"
+  go run ./test/conformance/walk.go --version="$DOC_VERSION" \
+                                  --url https://github.com/kubernetes/kubernetes/tree/release-"${DOC_VERSION}"/ \
+                                  --docs ./_output/specsummaries.json > ./_output/KubeConformance-"${DOC_VERSION}".md
+  sed "s|tree/release-${DOC_VERSION}.*/test/e2e|tree/release-${DOC_VERSION}/test/e2e|g" ./_output/KubeConformance-"${DOC_VERSION}".md > "${GITHUB_WORKSPACE}"/docs/KubeConformance-"${DOC_VERSION}".md
+  echo "${RELEASE_VERSION}" > /tmp/release-version
+  echo "Conformance test suite documention for $(cat /tmp/release-version) has been created"
+}
+
+function main() {
+  echo "Executing script: ${GITHUB_WORKSPACE}/docs/${BASH_SOURCE[0]}"
```

Comment:
Getting the error
```
./docs/check-conformance-test-suite-docs.sh: line 70: GITHUB_WORKSPACE: unbound variable
```

you can set this to `${GITHUB_WORKSPACE:-}` to default it to nothing if it's undeclared

Reply:
Fixed